File: java/funciones.java
public class Main {

    public static void saludar() {
        System.out.println("¡Hola, bienvenido al programa!");
    }

    public static int sumar(int a, int b) {
        return a + b;
    }

    public static float multiplicar(float x, float y) {
        return x * y;
    }

    public static void main(String[] args) {
        
        saludar();

        
        int resultadoSuma = sumar(5, 3);
        System.out.println("La suma de 5 y 3 es: " + resultadoSuma);

        
        float resultadoMultiplicacion = multiplicar(4.5f, 2.3f);
        System.out.println("El producto de 4.5 y 2.3 es: " + resultadoMultiplicacion);
    }
}

File: cpp/funciones.cpp
#include <iostream>
using namespace std;

// Función sin retorno (void) que imprime un mensaje
void saludar()
{
    cout << "¡Hola, bienvenido al programa!" << endl;
}

// int  toma dos parámetros enteros y devuelve su suma
int sumar(int a, int b)
{
    return a + b;
}

// Ffloat toma dos números flotantes y devuelve su producto
float multiplicar(float x, float y)
{
    return x * y;
}

int main()
{
    // Llamar a la función saludar
    saludar();

    // Llamar a la función sumar y mostrar el resultado
    int resultadoSuma = sumar(5, 3);
    cout << "La suma de 5 y 3 es: " << resultadoSuma << endl;

    // Llamar a la función multiplicar y mostrar el resultado
    float resultadoMultiplicacion = multiplicar(4.5, 2.3);
    cout << "El producto de 4.5 y 2.3 es: " << resultadoMultiplicacion << endl;

    return 0;
}
